content/tutorials/aws-persistence.textile

```diff
@@ -0,0 +1,249 @@
+---
+authors:
+- author_bio: ''
+  author_image: 
+  author_name: Steven Applebly
+  author_profile_url: 
+category:
+- integrations
+date_published: '2016-07-22T02:38:02+01:00'
+excerpt: Learn how to persist messages into AWS S3.
+group: sdk
+index: 10
+languages:
+- javascript
+last_updated: '2021-11-19T16:35:45+00:00'
+level: easy
+platform: mixed
+reading_time: 5 — 10
+section: tutorials
+tags:
+- Integrations
+- AWS Lambda
+title: AWS Persistence in S3
```

Comment:
I think the title needs changing. Something like:

"How to persist Ably messages to AWS S3"